nuxeo-core/nuxeo-core-bulk/src/main/resources/OSGI-INF/bulk-config.xml

```diff
@@ -27,11 +27,13 @@
 
   <!-- Defines some actions -->
   <extension target="org.nuxeo.ecm.core.bulk" point="actions">
-    <action name="setSystemProperties" bucketSize="100" batchSize="25" />
-    <action name="setProperties" bucketSize="100" batchSize="25" />
+    <!-- internal / non exposed -->
     <action name="removeProxy" bucketSize="100" batchSize="25" />
-    <action name="csvExport" bucketSize="100" batchSize="50" />
-    <action name="trash" bucketSize="100" batchSize="25" />
+    <action name="setSystemProperties" bucketSize="100" batchSize="25" />
+    <!-- actions available through REST API -->
+    <action name="setProperties" bucketSize="100" batchSize="25" exposed="true" />
+    <action name="csvExport" bucketSize="100" batchSize="50" exposed="true"/>
+    <action name="trash" bucketSize="100" batchSize="25" exposed="true"/>
```

Comment:
only internal action should be explicitly restricted, so the default should be exposed=true

Reply:
I thought the white list approach would be safer, don't you think so ?

Reply:
+1 on the whitelist approach